test/e2e/e2e_test.go

```diff
@@ -2,32 +2,37 @@ package e2e
 
 import (
 	"bytes"
+	"crypto/tls"
 	"crypto/x509"
 	"encoding/pem"
 	"fmt"
 	"math/rand"
+	"net/http"
 	"os"
+	"strconv"
 	"strings"
 	"testing"
 	"time"
 
-	v1 "k8s.io/api/core/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	kruntime "k8s.io/apimachinery/pkg/runtime"
-	"k8s.io/apimachinery/pkg/util/wait"
-	"k8s.io/client-go/kubernetes"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/tools/clientcmd"
-
+	routev1 "github.com/openshift/api/route/v1"
 	operatorv1client "github.com/openshift/client-go/operator/clientset/versioned"
+	routeclient "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
 	"github.com/openshift/library-go/pkg/crypto"
 	"github.com/openshift/library-go/pkg/operator/events"
 	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
 	"github.com/openshift/service-ca-operator/pkg/controller/api"
 	"github.com/openshift/service-ca-operator/pkg/operator"
 	"github.com/openshift/service-ca-operator/pkg/operator/operatorclient"
 	"github.com/openshift/service-ca-operator/test/util"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/util/intstr"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
```

Comment:
keep these in a separate block as before

Reply:
I don't know how to do it. It's automatically rearranged by the `VSCode` editor.

Reply:
I can't do that due to below error:
https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_service-ca-operator/81/pull-ci-openshift-service-ca-operator-master-verify/174

Reply:
```
diff --git a/test/e2e/e2e_test.go b/test/e2e/e2e_test.go
index 33d327b8..8389cc41 100644
--- a/test/e2e/e2e_test.go
+++ b/test/e2e/e2e_test.go
@@ -14,6 +14,16 @@ import (
        "testing"
        "time"
 
+       v1 "k8s.io/api/core/v1"
+       "k8s.io/apimachinery/pkg/api/errors"
+       metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+       kruntime "k8s.io/apimachinery/pkg/runtime"
+       "k8s.io/apimachinery/pkg/util/intstr"
+       "k8s.io/apimachinery/pkg/util/wait"
+       "k8s.io/client-go/kubernetes"
+       "k8s.io/client-go/rest"
+       "k8s.io/client-go/tools/clientcmd"
+
        routev1 "github.com/openshift/api/route/v1"
        operatorv1client "github.com/openshift/client-go/operator/clientset/versioned"
        routeclient "github.com/openshift/client-go/route/clientset/versioned/typed/route/v1"
@@ -24,15 +34,6 @@ import (
        "github.com/openshift/service-ca-operator/pkg/operator"
        "github.com/openshift/service-ca-operator/pkg/operator/operatorclient"
        "github.com/openshift/service-ca-operator/test/util"
-       v1 "k8s.io/api/core/v1"
-       "k8s.io/apimachinery/pkg/api/errors"
-       metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-       kruntime "k8s.io/apimachinery/pkg/runtime"
-       "k8s.io/apimachinery/pkg/util/intstr"
-       "k8s.io/apimachinery/pkg/util/wait"
-       "k8s.io/client-go/kubernetes"
-       "k8s.io/client-go/rest"
-       "k8s.io/client-go/tools/clientcmd"
 )
 
 const (

```
works just fine for me even in VS Code.

Reply:
Fixed